### PR TITLE
feature: Add button on public climber page to message them directly

### PR DESCRIPTION
**Pull request by tohhongxiang123**
Screenshot 2022-12-02 at 1 16 42 AM
Screenshot 2022-12-02 at 1 16 54 AM
Screenshot 2022-12-02 at 1 16 59 AM

Add a "message" button to allow user to message another user on telegram
- Does not show on your own profile page

### CODE REVIEW

**Comment by tohhongxiang123** [comment]
Fixed. 
Screenshot 2022-12-09 at 9 19 34 PM
Screenshot 2022-12-09 at 9 20 04 PM
Screenshot 2022-12-09 at 9 20 33 PM

Starts to break under 275px width though

File: frontend/src/components/profile/ProfileHeader.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ type ProfileHeaderProps = {
 
 export default function ProfileHeader({ user, loading = false }: ProfileHeaderProps) {
   return (
-    <Stack direction="row" alignItems="center">
+    <Stack direction="row" alignItems={"center"}>
```
Comment (CrownKira) [inline frontend/src/components/profile/ProfileHeader.tsx:13]: i suggest keeping it as "center" (removing {}) to be consistent with others

Comment (CrownKira) [reply frontend/src/components/profile/ProfileHeader.tsx:13]: similar issues below too, please rectify accordingly

**Review by CrownKira — CHANGES_REQUESTED** [review]

File: frontend/src/components/profile/ProfileHeader.tsx (hunk 1 above)
Comment (tohhongxiang123) [reply frontend/src/components/profile/ProfileHeader.tsx]: Ok, done. Thanks

**Review by CrownKira — APPROVED** [review]
LGTM!

**Pull request merged**